Assets/MRTK/Core/Inspectors/Setup/MixedRealityProjectConfiguratorWindow.cs

```diff
@@ -194,6 +199,7 @@ private void RenderConfigurations()
                     RenderToggle(MRConfig.SpatialPerceptionCapability, "Enable Spatial Perception Capability");
 #if UNITY_2019_3_OR_NEWER
                     RenderToggle(MRConfig.EyeTrackingCapability, "Enable Eye Gaze Input Capability");
+                    RenderToggle(MRConfig.GraphicsJobWorkaround, "Avoid Unity 'PlayerSettings.graphicsJob' crash");
```

Comment:
If there's a way to put a link in here, do we want to have a link to the issue in the comment above (https://github.com/microsoft/MixedRealityToolkit-Unity/issues/8326) for more info about the crash?

Reply:
I think it would be possible to add a button that goes to the issue - I think that I might actually go with the existing flow of having this documented here: https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/MRTK_Configuration_Dialog.html

Reply:
I think it would be possible to add a button that goes to the issue - I think that I might actually go with the existing flow of having this documented here: https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/MRTK_Configuration_Dialog.html